src/components/AddItem.js

```diff
@@ -1,7 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { writeToFirestore } from '../lib/firebase';
 
 const AddItem = () => {
-  return <h1>THIS IS THE ADD ITEM</h1>;
+  let [name, setName] = useState('');
+  let [frequency, setFrequency] = useState('');
```

Comment:
I see that the AC ask for the `frequency` value to be saved as an integer, but it is currently being saved as a string. you may need to `parseInt(frequency)` when you are writing to firestore in the handleSubmitForm. Note of caution, make sure there is a default value, just in case, otherwise JavaScript will get mad when you try to `parseInt` nothing.

Reply:
Maybe you could set the default value to `7` in the `useState` above, too...I think the wireframes ask for a default value.
https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/add-item.png

Reply:
![image](https://user-images.githubusercontent.com/10378331/90301356-fc58df00-de64-11ea-8475-eeed0acadcf8.png)
Firestore db is telling me it's being saved as a string.

Reply:
Updated to be sent as an integer to firestore!

Reply:
I was trying to have the first radio button selected by default by including the checked attribute, but it resulted in some funky behavior.  When I tried to switch between the other two radio buttons, it would jump back to the first radio button.  Do you guys know how to adjust it?  Is this something to do with fieldset?

Reply:
I was able to do it in a very hacky way (conditional rendering of input field), but would love any suggestions of a better way to fix this!

Reply:
@laurenyz I did a draft PR as an example: https://github.com/the-collab-lab/tcl-10-smart-shopping-list/pull/26

Reply:
Great catch with the frequency number needing to be saved as an integer. I believe the importance of this difference comes into play a little further down the line, once we begin implementing comparisons between the items and their various purchase frequencies.